src/impl/device/rotarysensor/IMU.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * @author Alex Riensche, UNL
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#include "okapi/impl/device/rotarysensor/IMU.hpp"
+#include "okapi/api/util/mathUtil.hpp"
+
+namespace okapi {
+IMU::IMU(const std::uint8_t iport, const IMUAxes iaxis) : port(iport), axis(iaxis) {
+}
+
+IMU::~IMU() = default;
+
+double IMU::get() const {
+  double angle = readAngle() - offset;
+
+  if (angle == PROS_ERR) {
+    return PROS_ERR;
+  }
+
+  if (angle > 180 || angle < -180) {
```

Comment:
This functionality matches the recently added okapilib helper functions to constrain an angle to 180 degrees. Perhaps use those instead?